simple-facebook-instant-articles.php

```diff
@@ -174,8 +174,8 @@ public function customise_feed_query( $query ) {
 		// Customise FB IA feed query.
 		if ( $query->is_main_query() && $query->is_feed( $feed_slug ) ) {
 
-			$query->set( 'posts_per_rss', 25 );
-			$query->set( 'orderby', 'modified' );
+			$num_posts = intval( apply_filters( 'simple_fb_posts_per_rss', get_option( 'posts_per_rss', 10 ) ) );
+			$query->set( 'posts_per_rss', $num_posts );
 
```

Comment:
Oh will have been a merge conflict... good catch. I'll just push a revert commit straight to master